astroid/context.py

```diff
@@ -10,11 +10,14 @@
 if TYPE_CHECKING:
     from astroid.nodes.node_classes import Keyword, NodeNG
 
+_InferenceCache = MutableMapping[
```

Comment:
Any idea why it was `MutableMapping` instead of `Dict`?

Do we want to start adding a `Type` suffix to type aliases?

Reply:
PR that added it is https://github.com/PyCQA/astroid/pull/1009.

Perhaps adding `Type` is something to do after we can run `mypy` successfully? I'm still running into issues with types not actually being correct or missing information. I think we can do a "cleanup" `mypy` PR after we can get it to pass.

Reply:
> PR that added it is #1009.

Let's use `Dict`. Usually the return type should be precise whereas the parameter type should be open. `Sequence["NodeNG"]` is ok though I think. We assign a list converted to a tuple to it.

> Perhaps adding `Type` is something to do after we can run `mypy` successfully? I'm still running into issues with types not actually being correct or missing information. I think we can do a "cleanup" `mypy` PR after we can get it to pass.

👍🏻